### PR TITLE
Adding Acknowledgements in README

### PR DESCRIPTION
**Pull request by jessikra**
We should all add some feedback here

### CODE REVIEW

**Review by jessikra — COMMENTED** [review]
Please add feedback

**Review by JJaime1996 — COMMENTED** [review]
Hello! Can you see this?

**Comment by Danny393** [comment]
Looks good so far

File: README.md [hunk 1]
```diff
@@ -47,4 +47,5 @@ Make sure you have Homebrew installed
 Your Name - [@your_twitter](https://twitter.com/your_username) - email@example.com
 
 <!-- ACKNOWLEDGEMENTS -->
-## Acknowledgements
\ No newline at end of file
+## Acknowledgements
+<We need to add Project Owner and team info>
```
Comment (Danny393) [inline README.md]: Make sure to remember to fill in all the placeholder values near the end of the project

**Comment by dailenperezFIU** [comment]
Looks good to me :)

**Review by OAKevin — CHANGES_REQUESTED** [review]
Should we use https://expo.dev/ as the expo website?

File: README.md [hunk 2]
```diff
@@ -47,4 +47,5 @@ Make sure you have Homebrew installed
 Your Name - [@your_twitter](https://twitter.com/your_username) - email@example.com
```
Comment (rlope265) [inline README.md:58]: Is this section for everyone in the team to put their contact information?

**Comment by armand737** [comment]
It looks great!

**Review by OAKevin — APPROVED** [review]
Ignore my comment I tested it and Its working fine on my laptop

**Pull request closed without merging**